examples/logs-basic/src/main.rs

```diff
@@ -0,0 +1,27 @@
+use log::{error, Level};
```

Comment:
Can we move this example to `opentelemetry-appender-log` folder? The examples is used to demonstrate the how to use this bridge after all and we already have too many examples in top-level example folder

Reply:
Though I agree it doesn't look good to have too many top-level examples, I believe this one is best served in the top-level.
1. It should be okay to have a basic "hello world" style example for each of the signals (metrics, traces already exist) in the top level folder.
2. I can do some cleanup of the existing top-level examples to trim them down.

Thoughts?